mono/mini/simd-intrinsics-netcore.c

```diff
@@ -332,10 +332,14 @@ emit_sys_numerics_vector_t (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSig
 			opcode = OP_XEXTRACT_R4;
 			dreg = alloc_freg (cfg);
 			break;
-		default:
+		case MONO_TYPE_I4:
+		case MONO_TYPE_U4:
 			opcode = OP_XEXTRACT_I32;
 			dreg = alloc_ireg (cfg);
 			break;
+		default:
+			// FIXME:
+			return NULL;
```

Comment:
This still leaves out the exception checks generated above. I haven't checked if they get optimized out.